C4SGWeb/src/app/user/profile/user-profile.component.ts

```diff
@@ -18,7 +21,16 @@ export class UserProfileComponent {
     skills: new FormControl('', Validators.required)
   });
 
-  constructor() { }
+  ngAfterViewInit() {
+
+     $('select').material_select();
```

Comment:
use the Materialize directive instead of jQuery

https://github.com/InfomediaLtd/angular2-materialize/blob/master/app/components/forms.ts#L65